new-post-content-process (#113)
NHSDigital/prevention-services-design-history: app/posts/select-people-for-invitation/2025-04-16-content-process.md

---

> title: How we're developing SPI content 

Reviewer: Generally I think we don’t need to mention the service name in the post titles (even abbreviated) as it’d get repetitive and _should_ be clear from the context. (If not we can make it clearer).

Could instead be a bit more specific about what you mean by 'content' here?

```suggestion
title: How we manage our user interface content
```

Author: updated

description: Our approach to content design and testing
date: 2025-04-16
tags:
  - service
  - content
  - user research
---

> ## Background 

Reviewer: Could drop this first h2, as it’s kinda implied that the opening paragraph gives a background or overview, and going straight from an h1 to an h2 can look a bit odd?

```suggestion
```

Author: updated

> Before SPI was built, campaigns were created with configuration files (JSON format) and largely used technical language. Initially, much of this terminology was copied into the UI but we simplified the language as we built SPI and continue to iterate. 

Reviewer: ```suggestion
Before this service was built, campaigns were created with configuration files (JSON format) and largely used technical language. Initially, much of this terminology was copied into the user interface but we simplified the language as we built the service and continue to iterate. 
```

Author: updated

Our goal is to empower all users – both current expert users, and potential new non-expert users. This aligns to our metric of making SPI easier to understand and simplifying the experience.

## Challenges

Our current SPI users, the Operational team, are experts, so they are comfortable with the technical language used. They are also a small group of users to test with. The team consists of:

- Delivery lead
- Delivery support
- Technical lead
- Technical support

During user research sessions, our expert users confirmed our hypothesis that a lot of the language would not make sense to any new users coming on board.

From the outset, we've been working on making the content clearer for novice users and we’re continually updating the content to align with new features, technical updates, and insights from user research sessions. 

## How we track content changes

> To get SPI ready for MVP, we created a content board in Mural. This helps us to:

Reviewer: ```suggestion
To get the service ready as a minimal viable product (MVP), we created a content board in Mural. This helps us to:
```

Author: good shout - updated

- track and share content changes
- keep a record of ideas, feedback and decisions

We've continued using the Mural board as our main content document. At each stage or release, we add a new block to show the latest updates.

To keep everything accurate, we take screenshots from the live SPI environment and highlight any new or changed content in red. We then cross-check these against what's live in SPI, so we always have a single, up-to-date source of truth.

[![zoomed out view of a Mural board with screen shots](SPI-content-mural.png)](SPI-content-mural.png)

## Mural board and content process

As we've progressed through MVP, Core (Alpha) and Smart (Beta) the content has continued to evolve.

We update it regularly based on:

- user research sessions
- new features
- technical updates
- releases

> Example of captured UR insights:

Reviewer: ```suggestion
Example of insights from user research:
```

Author: good shout - updated

> [![Snippet of a Mural board showing some examples of content insights](SPI-content-insights.png)](SPI-content-insights.png)

Reviewer: Not sure the screenshot is adding much here? Could convert to a list instead? (Would be more accessible)

Author: swapped to bullets

[![Example of user feedback on content and terminology](SPI-content-asynchronous.png)](SPI-content-asynchronous.png)

We collate the answers anonymously, which helps users feel more comfortable giving honest feedback. Example: 

[![Example of user feedback collated in a Mural board](SPI-content-collate-results.png)](SPI-content-collate-results.png)

We then sort the feedback into 'suggested alternatives' and 'difficulty to explain' ratings. Example:

[![Example of user feedback with scored difficulty ratings](SPI-content-difficulty-rating.png)](SPI-content-difficulty-rating.png)

## Content insights

We structure user research sessions to focus on specific parts of SPI, which helps us gather targeted feedback on how content is working.

One key insight is that the language the Operational team uses day-to-day often doesn’t match the technical terms in SPI.

### Example 1: 'Copying iterations'

We originally used the term 'duplicating iterations', but users told us that 'copy' or 'clone' felt more natural. These are phrases they're familiar with from everyday tasks, like 'copy and paste' or 'copy a file'. Based on this, we're updating the language to reflect the way they talk:

> [![Screen shot showing why we changed the phrase duplicate iterations to copy iterations](SPI-content-copying-iterations.png)](SPI-content-copying-iterations.png)

Reviewer: Nice example!

Author: 👍 thumbs
